### ISSUE TITLE
Can't scroll back up on ListView without triggering refresh

### ISSUE THREAD

**Issue by jennaharris7** [iskugor/Ti.SwipeRefreshLayout#15]
When using a listview, if a user scrolls down the list, and then attempts to scroll back up, the refresh action is initiated again immediately

**Comment by aggelos81** [comment]
In which version of android are you seeing this issue, because I cannot recreate it with a simple example like this

```javascript
var swiperefreshlayout = require('com.rkam.swiperefreshlayout');
var listView = Ti.UI.createListView();
var sections = [];

var fruitSection = Ti.UI.createListSection({ headerTitle: 'Fruits'});
var fruitDataSet = [
    {properties: { title: 'Apple'}},
    {properties: { title: 'Banana'}},
    {properties: { title: 'Apple'}},
    {properties: { title: 'Banana'}},
    {properties: { title: 'Apple'}},
    {properties: { title: 'Banana'}},
    {properties: { title: 'Apple'}},
    {properties: { title: 'Banana'}},
    {properties: { title: 'Apple'}},
    {properties: { title: 'Banana'}},
    {properties: { title: 'Apple'}},
    {properties: { title: 'Banana'}}
];
fruitSection.setItems(fruitDataSet);
sections.push(fruitSection);

var vegSection = Ti.UI.createListSection({ headerTitle: 'Vegetables'});
var vegDataSet = [
    {properties: { title: 'Carrots'}},
    {properties: { title: 'Potatoes'}},
    {properties: { title: 'Carrots'}},
    {properties: { title: 'Potatoes'}},
    {properties: { title: 'Carrots'}},
    {properties: { title: 'Potatoes'}},
    {properties: { title: 'Carrots'}},
    {properties: { title: 'Potatoes'}},
    {properties: { title: 'Carrots'}},
    {properties: { title: 'Potatoes'}},
    {properties: { title: 'Carrots'}},
    {properties: { title: 'Potatoes'}}
];
vegSection.setItems(vegDataSet);
sections.push(vegSection);

listView.sections = sections;

var fishSection = Ti.UI.createListSection({ headerTitle: 'Fish'});
var fishDataSet = [
    {properties: { title: 'Cod'}},
    {properties: { title: 'Haddock'}},
    {properties: { title: 'Cod'}},
    {properties: { title: 'Haddock'}},
    {properties: { title: 'Cod'}},
    {properties: { title: 'Haddock'}},
    {properties: { title: 'Cod'}},
    {properties: { title: 'Haddock'}},
    {properties: { title: 'Cod'}},
    {properties: { title: 'Haddock'}},
    {properties: { title: 'Cod'}},
    {properties: { title: 'Haddock'}},
    {properties: { title: 'Cod'}},
    {properties: { title: 'Haddock'}}
];
fishSection.setItems(fishDataSet);
listView.appendSection(fishSection);
var swipeRefresh = swiperefreshlayout.createSwipeRefresh({
    view: listView,
    height: Ti.UI.FILL,
    width: Ti.UI.FILL
});
win.add(swipeRefresh);
```

**Comment by ilker55** [comment]
I'm having the same issue.
I have a home.js which creates a scrollableview. And one of the views is search.js which has a listview made in alloy.

home.js:
```javascript
Ti.App.tabGroup = Ti.UI.createScrollableView({
  	views: [
  		...
  		Alloy.createController('search').getView(),
  		...
	],
	  ...
});
```
search.xml:
```xml
<Alloy>
	<Window id="window">
		<View id="viewProducts">
			<Widget id="ptr" src="nl.fokkezb.pullToRefresh" onRelease="myRefresher">
				<ListView id="lvProducts" defaultItemTemplate="default">
					...
				</ListView>
			</Widget>
			<Label id="lbNoLocation" />
		</View>
	</Window>
</Alloy>
```

**Comment by aggelos81** [comment]
But you are using this widget https://github.com/FokkeZB/nl.fokkezb.pullToRefresh while reporting an issue on this widget https://github.com/iskugor/Ti.SwipeRefreshLayout



—

**Comment by ilker55** [comment]
@aggelos81 https://github.com/FokkeZB/nl.fokkezb.pullToRefresh uses com.rkam.swiperefreshlayout like yours, so i thought it was the same.

**Comment by ilker55** [comment]
Btw I fixed it by updating to version 0.5.4 https://github.com/iskugor/Ti.SwipeRefreshLayout/blob/master/dist/com.rkam.swiperefreshlayout-android-0.5.4.zip. Maybe it will work for @jennaharris7 too.

**Comment by iskugor** [comment]
This seems to be all right with latest module.
